File: Backend/Java env/auth-service/Eclothing/src/main/java/com/example/demo/dto/SellerProfileResponse.java
package com.example.demo.dto;

public class SellerProfileResponse {
	private Integer userId;
    private String name;
    private String email;
    private String mobile;

    private String shopName;
    private String gstNumber;
    private String businessAddress;
    
	public SellerProfileResponse() {
	}

	public SellerProfileResponse(Integer userId, String name, String email, String mobile, String shopName,
			String gstNumber, String businessAddress) {
		super();
		this.userId = userId;
		this.name = name;
		this.email = email;
		this.mobile = mobile;
		this.shopName = shopName;
		this.gstNumber = gstNumber;
		this.businessAddress = businessAddress;
	}

	public Integer getUserId() {
		return userId;
	}

	public void setUserId(Integer userId) {
		this.userId = userId;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getMobile() {
		return mobile;
	}

	public void setMobile(String mobile) {
		this.mobile = mobile;
	}

	public String getShopName() {
		return shopName;
	}

	public void setShopName(String shopName) {
		this.shopName = shopName;
	}

	public String getGstNumber() {
		return gstNumber;
	}

	public void setGstNumber(String gstNumber) {
		this.gstNumber = gstNumber;
	}

	public String getBusinessAddress() {
		return businessAddress;
	}

	public void setBusinessAddress(String businessAddress) {
		this.businessAddress = businessAddress;
	}
}

File: Backend/DotNet env/AuthService/Models/DTOs/SellerProfileResponse.cs
namespace AuthService.Models.DTOs
{
    public class SellerProfileResponse
    {
        public int UserId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Mobile { get; set; } = string.Empty;

        public string ShopName { get; set; } = string.Empty;
        public string GstNumber { get; set; } = string.Empty;
        public string BusinessAddress { get; set; } = string.Empty;

        public SellerProfileResponse() { }

        public SellerProfileResponse(int userId, string name, string email, string mobile, string shopName, string gstNumber, string businessAddress)
        {
            UserId = userId;
            Name = name;
            Email = email;
            Mobile = mobile;
            ShopName = shopName;
            GstNumber = gstNumber;
            BusinessAddress = businessAddress;
        }
    }
}
